server/runtime_javascript_nakama.go

```diff
@@ -5019,6 +5020,58 @@ func (n *runtimeJavascriptNakamaModule) leaderboardsGetId(r *goja.Runtime) func(
 	}
 }
 
+// @group Leaderboards
+// @summary Fetch the list of leaderboard records around the owner.
+// @param id(type=string) The unique identifier for the leaderboard.
+// @param owner(type=string) The owner of the score to list records around. Mandatory field.
+// @param limit(type=number) Return only the required number of leaderboard records denoted by this limit value.
+// @param overrideExpiry(type=number) Records with expiry in the past are not returned unless within this defined limit. Must be equal or greater than 0.
+// @return records(nkruntime.LeaderboardRecordList) The leaderboard records according to ID.
+// @return error(error) An optional error value if an error occurred.
+func (n *runtimeJavascriptNakamaModule) leaderboardRecordsAroundOwner(r *goja.Runtime) func(goja.FunctionCall) goja.Value {
+	return func(f goja.FunctionCall) goja.Value {
+		id := getJsString(r, f.Argument(0))
+		if id == "" {
+			panic(r.NewTypeError("expects a leaderboard ID string"))
+		}
+
+		ownerID := getJsString(r, f.Argument(1))
+		uid, err := uuid.FromString(ownerID)
+		if err != nil {
+			panic(r.NewTypeError("expects user ID to be a valid identifier"))
+		}
+
+		limit := 10
+		if f.Argument(2) != goja.Undefined() && f.Argument(2) != goja.Null() {
+			limit = int(getJsInt(r, f.Argument(2)))
+			if limit < 1 || limit > 100 {
+				panic(r.NewTypeError("limit must be 1-100"))
+			}
+		}
+
+		overrideExpiry := int64(0)
+		if f.Argument(4) != goja.Undefined() {
```

Comment:
I think this should be
```suggestion
		if f.Argument(3) != goja.Undefined() {
```

Reply:
Good catch, updating now